src/scripts/lightningLinkInterceptor.js

```diff
@@ -9,13 +9,17 @@ function interceptClickEvent(e) {
             window.postMessage(JSON.stringify({"lightningLink": url}), "*");
         }
     }
-    // if (target.tagName === 'A') {
-    //     href = target.getAttribute('href');
-    //     console.log("interceptClickEvent href=" + href);
-    //     if (href && href.startsWith("lightning:")) {
-    //         window.postMessage(JSON.stringify({"lightningLink": href}), "*");
-    //     }
-    // }
+    if (target.tagName === 'A') {
```

Comment:
Can we simplify this method by just querying 2-3 levels for parents that contains lightning link? like a simple loop instead of these conditions? Maybe also we don't care if the child is 'svg' or 'rect'?

Reply:
We can replace it with this:
```
target.closest("a")
```
but it's likely to prompt wrong/unwanted invoices as it can go above 2-3 levels.

We can run this for 3 layers with a counter:
```     
var counter = 0;
while(target.parentElement && counter < 2) {
    target = target.parentElement;
    if(target.hasAttribute('href')) {
        href = target.getAttribute('href');
        break;
    }
    counter++;
}
```
or we can inject css to disable clicks on SVG
```
((d,e)=>{
  e = d.createElement("style");
  e.innerHTML = "svg{pointer-events: none;}";
  d.head.appendChild(e)
})(document)
```

Reply:
@erdemyerebasmaz disabling svg may disable non lightning links as well right?

Reply:
@roeierez Not likely, we are disabling clicks on svg element, if it is surrounded by an element that has a link( <a href=""> or <button> with onclick etc.) it will follow through that link.